perf_test/blas/blas3/KokkosBlas3_gemm_perf_test.hpp

```diff
@@ -465,6 +467,89 @@ void __do_gemm_serial_batched(options_t options, gemm_args_t gemm_args) {
   return;
 }
 
+template <class algo_tag, class blocking_type, class device_type,
+          class algo_mode = void>
+void __do_batchedGemm_parallel(options_t options, gemm_args_t gemm_args) {
+  BatchedGemmHandle batchedGemmHandle(BaseHeuristicAlgos::SQUARE);
```

Comment:
Same as above. Is this for testing square matrix with heuristic algorithm? Should we rename the function name to, perhaps, ```__do_batchedGemmHeuristic_parallel```?